src/app/vendors/d2-vendors.ts

```diff
@@ -225,9 +226,10 @@ export function filterVendorGroups(
 }
 
 export function filterToUnacquired(ownedItemHashes: Set<number>): VendorFilterFunction {
-  return ({ owned, item, collectibleState }) =>
+  return ({ owned, item, collectibleState, failureStrings }) =>
     item &&
     !owned &&
+    !(failureStrings.includes(t('Vendors.AlreadyAcquired')) && item.isExotic) &&
```

Comment:
We can't do this sort of thing - depending on localized strings is too hard to keep working